### PR TITLE
Cleanup code

### PR DESCRIPTION
**Pull request by andriyParashchuk**
Need move all static text to YAML files

### CODE REVIEW

File: _data/customers.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+customer_1:
+  name: 'John Williams'
+  position: 'Director of Product'
+  text: 'Once configured it just works and saves ton of time. Allows you to keep critical data in Airtable and import into google sheets for analytics and calculations. Once configured it just works and saves ton of time. Allows you to keep critical data in Agent Inbox.'
```
Comment (obliviusm) [inline _data/customers.yml]: can you make it multiline instead of one long line? https://stackoverflow.com/a/21699210

**Review by obliviusm — CHANGES_REQUESTED** [review]
pls check comment
otherwise good

**Comment by andriyParashchuk** [comment]
@obliviusm Done

**Comment by andriyParashchuk** [comment]
@obliviusm please, review again

**Review by obliviusm — APPROVED** [review]
LGTM
pls prepare commits and merge

**Comment by obliviusm** [comment]
Deployed to https://jetthoughts-pr-371.surge.sh

**Pull request closed without merging**